lib/web_console/templates/console.js.erb

```diff
@@ -54,11 +54,34 @@ var styleElementId = 'sr02459pvbvrmhco';
 
 // REPLConsole Constructor
 function REPLConsole(config) {
+  function getConfig(key, defaultValue) {
+    return config && config[key] || defaultValue;
+  }
+
   this.commandStorage = new CommandStorage();
-  this.prompt = config && config.promptLabel ? config.promptLabel : ' >>';
-  this.commandHandle = config && config.commandHandle ? config.commandHandle : function() { return this; }
+  this.prompt = getConfig('promptLabel', ' >>');
+  this.remotePath = getConfig('remotePath');
+  this.request = getConfig('request', this.request);
 }
 
+REPLConsole.prototype.request = request;
+
+REPLConsole.prototype.getUrl = function(path) {
+  if (path) {
+    return this.remotePath + '/' + path;
+  } else {
+    return this.remotePath;
+  }
+};
+
+REPLConsole.prototype.commandHandle = function(line) {
+  var _this = this;
+  var params = 'input=' + encodeURIComponent(line);
+  this.request('PUT', this.getUrl(), params, function(xhr) {
```

Comment:
What happens when this request fails? I think we should handle both cases, showing an appropriate message when this is failing.